charts/grafana/templates/configmap.yaml

```diff
@@ -13,7 +13,8 @@ data:
 {{- range $key, $value := index .Values "grafana.ini" }}
     [{{ $key }}]
     {{- range $elem, $elemVal := $value }}
-    {{ $elem }} = {{ $elemVal }}
+    {{ $elem }} : |
+  {{ tpl (toYaml $value | indent 4) }}
```

Comment:
Error:  templates/: template: grafana/templates/deployment.yaml:34:28: executing "grafana/templates/deployment.yaml" at <include (print $.Template.BasePath "/configmap.yaml") .>: error calling include: template: grafana/templates/configmap.yaml:17:5: executing "grafana/templates/configmap.yaml" at <tpl>: wrong number of args for tpl: want 2 got 1